junit-commons/src/main/java/org/junit/gen5/commons/util/Preconditions.java

```diff
@@ -67,6 +72,7 @@ public static void notEmpty(String str, String message) throws IllegalArgumentEx
 	 * @param str the string to check
 	 * @param messageSupplier precondition failure message supplier
 	 * @see StringUtils#isNotEmpty(CharSequence)
+	 * @throws IllegalArgumentException if the string is {@code null} or empty
```

Comment:
There's a Checkstyle check that fails now:
```
:junit-commons:checkstyleMain[ant:checkstyle] /home/travis/build/junit-team/junit-lambda/junit-commons/src/main/java/org/junit/gen5/commons/util/Preconditions.java:75: error: At-clauses have to appear in the order '[@param, @return, @throws, @exception, @author, @since, @see]'.
```

The tag `@throws` should be above `@see`.